src/xtopcom/xbasic/src/xbyte_buffer.cpp

```diff
@@ -63,5 +63,13 @@ random_base58_bytes(std::size_t const size) {
     return ret;
 }
 
+xbyte_buffer_t from_bytes(uint8_t* data, std::size_t size) {
```

Comment:
this API seems useless.  usually we use xbyte_buffer_t{ data, data + size } converting (uint8_t * data, size_t size) to xbyte_buffer_t.

Reply:
ok